### PR TITLE
Media migration: Migrate existing Media entities

### PR DESCRIPTION
**Pull request by Adnan-cds**
- Migrate all existing Media entities.
- When creating parent Media entities for all **existing** File entities, ignore those files who already have a parent Media entity.

This is a continuation of the previous File and Media migration work: #20 .  Here we implement Steve's final suggestion from #20 :
> An alternative solution would be to migrate existing media entities and then migrate the files into media entities. You would then need to add a check to see if a file was already referenced by a media entity as otherwise there would be multiple media entities for a single file.

### CODE REVIEW

File: config/install/migrate_plus.migration.media_existing.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+process:
+  mid: mid
+  vid: vid
+  status: status
+  uid: uid
+  uuid: uuid
+  name: name
+  created: created
+  changed: changed
+  bundle: bundle
```
Comment (stephen-cox) [inline config/install/migrate_plus.migration.media_existing.yml:21]: This migration is going to cause problems for BHCC. It assumes that the Media bundle already exists in the LocalGov site. This is fine for Croydon where the two media bundles in use match those in LocalGov (`document` and `media`), but the DB I have from BHCC has a `news_default_image` bundle.

Ideally, there should be a check that the bundle is one of those included with LocalGov, but perhaps @andybroomfield can say whether this is likely to be a problem for him.

Comment (Adnan-cds) [reply config/install/migrate_plus.migration.media_existing.yml:21]: Good point.  I will add a check.  This is why you need code reviews :)

Comment (Adnan-cds) [reply config/install/migrate_plus.migration.media_existing.yml]: I have just added a restriction for the imported bundle types.  Only the *image*, *document*, and *remote_video* bundles are now processed as these are [declared](https://github.com/localgovdrupal/localgov_core/tree/master/modules/localgov_media/config/optional) in the localgov_core:localgov_media submodule.

**Review by stephen-cox — APPROVED** [review]
I haven't had a chance to test the latest change, but it looks good to me

**Comment by Adnan-cds** [comment]
Thanks Steve.  I am merging... :)

**Pull request merged**